code/game/objects/items/storage/storage.dm

```diff
@@ -328,65 +328,65 @@
 	return
 
 //This proc return 1 if the item can be picked up and 0 if it can't.
-//Set the stop_messages to stop it from printing messages
-/obj/item/storage/proc/can_be_inserted(obj/item/W as obj, stop_messages = 0)
+//Set the warning to stop it from printing messages
+/obj/item/storage/proc/can_be_inserted(obj/item/W as obj, warning = TRUE)
 	if(!istype(W) || (W.flags_item & NODROP))
 		return //Not an item
 
-	if(src.loc == W)
+	if(loc == W)
 		return FALSE //Means the item is already in the storage item
 	if(storage_slots != null && contents.len >= storage_slots)
-		if(!stop_messages)
+		if(warning)
 			to_chat(usr, "<span class='notice'>[src] is full, make some space.</span>")
 		return FALSE //Storage item is full
 
-	if(can_hold.len)
-		var/ok = 0
+	if(length(can_hold))
+		var/ok = FALSE
 		for(var/A in can_hold)
-			if(istype(W, text2path(A) ))
-				ok = 1
+			if(istype(W, text2path(A)))
+				ok = TRUE
 				break
 		if(!ok)
-			if(!stop_messages)
-				if (istype(W, /obj/item/tool/hand_labeler))
-					return 0
+			if(warning)
+				if(istype(W, /obj/item/tool/hand_labeler))
+					return FALSE
```

Comment:
if i'm correct that this stops a hand labeller being put in a bag, then just make it return FALSE if the hand labeller is turned on.